contrib/fparser/fparser_ad.hh

```diff
@@ -122,7 +122,7 @@ private:
   bool JITCompileHelper(const std::string &);
 
   /// JIT function pointer
-  Value_t (*compiledFunction)(const Value_t *, const Value_t *, const Value_t);
+  Value_t (*compiledFunction)(const Value_t *, const Value_t *, const Value_t, Value_t *);
```

Comment:
The compiled functions now take a pointer to a stack rather than using a local array.